libs/db-browser/src/test/java/com/oceanbase/tools/dbbrowser/util/DBSchemaAccessors.java

```diff
@@ -79,23 +80,36 @@ private String parseObVersionComment(String obVersionComment) {
 
     public DBSchemaAccessor createOBMysql() {
         this.version = getOBVersion();
-        return DBSchemaAccessorGenerator.createForOBMySQL(this.jdbcTemplate, null,
-                this.version, null);
+        return DBBrowser.schemaAccessor()
+                .setDbVersion(this.version)
+                .setJdbcOperations(this.jdbcTemplate)
+                .setType(DBBrowserFactory.OB_MYSQL)
+                .create();
     }
 
     public DBSchemaAccessor createOBOracle() {
         this.version = getOBVersion();
-        return DBSchemaAccessorGenerator.createForOBOracle(this.jdbcTemplate,
-                this.version);
+        return DBBrowser.schemaAccessor()
+                .setDbVersion(this.version)
+                .setJdbcOperations(this.jdbcTemplate)
+                .setType(DBBrowserFactory.OB_ORACLE)
+                .create();
     }
 
     public DBSchemaAccessor createOracle() {
-        return DBSchemaAccessorGenerator.createForOracle(this.jdbcTemplate);
+        return DBBrowser.schemaAccessor()
+                .setJdbcOperations(this.jdbcTemplate)
+                .setType(DBBrowserFactory.ORACLE)
+                .create();
     }
 
     public DBSchemaAccessor createMysql() {
         this.version = getMySQLVersion();
-        return DBSchemaAccessorGenerator.createForMySQL(this.jdbcTemplate, this.version);
+        return DBBrowser.schemaAccessor()
```

Comment:
Should we add `createDoris()` here?

Reply:
Its just a test util, we may add a method based on our needs~ I prefer add `createDoris` when we really need it~